theme/colors.ts

```diff
@@ -1,8 +1,37 @@
-import { colors as leatherColors } from '@leather-wallet/tokens';
 import { defineTokens } from '@pandacss/dev';
 
 // ts-unused-exports:disable-next-line
 export const colors = defineTokens.colors({
   current: { value: 'currentColor' },
-  ...leatherColors,
+  overlay: { value: 'rgba(0,0,0,0.4)' },
+  // TEMPORARY FIX FOR BROKEN COLORS
+  // TODO: Remove when we have all use case semantic token colors`
```

Comment:
In the `containers` branch I am using a these to override the theme. 

I only use these ones so we could remove the rest from here?
```
lightModeInk.1
lightModeInk.12
darkModeInk.1
darkModeInk.12
```